modules/install/pages/install-ports.adoc

```diff
@@ -89,8 +89,29 @@ a| * Version 2 (XMEM)
 ** *Encrypted*: 11207, 18091, 18092
 
 NOTE: If enforcing TLS encryption, these ports may be blocked outside of a Couchbase Server cluster but need to remain open between nodes.
+
+| _cbbackupmgr (backup/restore client)_
+a| *Unencrypted*: 8091-8096, 9102, 11210
+
+*Encrypted*: 11207, 18091-18096, 19102
+
 |===
 
+.ports for `cbbackupmgr`
+****
+`cbbackupmgr`, the backup client, connects to the Couchbase Server using the ports listed above. You can find detailed information about the server ports in the xref:detailed-port-description[] section, but below is a summary of the services/info cbbackupmgr is accessing via the ports.
+
+[horizontal]
+rest_port / ssl_rest_port:: 8091 / 18091 (Cluster admin/management)
+cbas_http_por t/ cbas_ssl_port:: 8095 / 18095 (Analytics service)
+memcached_port / memcached_ssl_port:: 11210 / 11207 (Data service)
+eventing_http_port / eventing_ssl_port:: 8096/18096 (Eventing service)
+indexer_http_port / indexer_https_port:: 9102 / 19102 (GSI Indexes)
+query_port / ssl_query_port:: 8093 / 18093 (Query)
+fts_http_port / fts_ssl_port:: 8094 / 18094 (Search)
+capi_port / ssl_capi_port:: 8092 / 18092 (Views)
+****
+
```

Comment:
This section should be a in a Note box like the note under the XDCR (cluster-to-cluster).

Also, the text can probably be shortened to:

cbbackupmgr, the backup client, connects to the Couchbase Server using the ports listed above.  You can find detailed information about the server ports in the Detailed Port Description.  Below is a summary of the services cbbackupmgr is accessing via the ports.

rest_port / ssl_rest_port:: 8091 / 18091 (Cluster admin/mgmt)
cbas_http_port / cbas_ssl_port:: 8095 / 18095 (Analytics service)
memcached_port / memcached_ssl_port:: 11210 / 11207 (Data service)
eventing_http_port / eventing_ssl_port:: 8096/18096 (Eventing service)
indexer_http_port / indexer_https_port:: 9102 / 19102 (GSI Indexes)
query_port / ssl_query_port:: 8093 / 18093 (Query)
fts_http_port / fts_ssl_port:: 8094 / 18094 (Search)
capi_port / ssl_capi_port:: 8092 / 18092 (Views)